id: PYSEC-0000-CVE-2022-2996
severity:
- type: CVSS_V3
  score: CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:N
details: A flaw was found in the python-scciclient when making an HTTPS connection
  to a server where the server's certificate would not be verified. This issue opens
  up the connection to possible Man-in-the-middle (MITM) attacks.
affected:
- package:
    name: python-scciclient
    ecosystem: PyPI
    purl: pkg:pypi/python-scciclient
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "0"
references:
- type: FIX
  url: https://opendev.org/x/python-scciclient/commit/274dca0344b65b4ac113d3271d21c17e970a636c
- type: WEB
  url: https://opendev.org/x/python-scciclient/commit/274dca0344b65b4ac113d3271d21c17e970a636c
- type: ARTICLE
  url: https://lists.debian.org/debian-lts-announce/2022/11/msg00006.html
- type: WEB
  url: https://lists.debian.org/debian-lts-announce/2022/11/msg00006.html
aliases:
- CVE-2022-2996
modified: "2022-12-12T21:09:00Z"
published: "2022-09-01T18:15:00Z"
